src/update_agent/actor.rs

```diff
@@ -138,17 +144,48 @@ impl UpdateAgent {
         std::time::Duration::from_secs(secs.saturating_add(jitter))
     }
 
+    /// Log at INFO level how many and which deployments will be excluded from being
+    /// future update targets.
+    fn log_excluded_depls(deployments: Result<BTreeSet<Release>, ()>, actor: &UpdateAgent) {
+        if let Ok(depls) = deployments {
+            let excluded_depls_count = depls.len() - 1; // Minus current booted deployment.
+            LOCAL_DEPLOYMENTS_COUNT.set(excluded_depls_count.try_into().unwrap());
```

Comment:
Bunch of comments on this:
 * This should be periodically refreshed too, ideally close to the querying logic itself
 * Would a `as` work here instead?